### DISCUSSION TITLE
Bottomsheet problem

### DISCUSSION THREAD

**Discussion by lucky-sky-777** [Sketchware-Pro/Sketchware-Pro#674] [Q&A] [answered]
Edittext in bottomsheet unclickable

**Comment by j-px** [comment, marked as answer]
Sketchware Pro v6.4.0 reverted a change made with earlier Sketchware Pro versions so that ListView's onItemClick Event works again.

To fix a View in a Custom View not being clickable in e.g. a dialog, add `android:focusable="true"` to the View's inject property.
